src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -848,7 +847,7 @@ def load_arguments(self, _):
 
     with self.argument_context('network express-route port link update', min_api='2019-08-01') as c:
         c.argument('admin_state',
-                   arg_type=express_route_link_admin_state_type,
```

Comment:
If `express_route_link_admin_state_type` will not use in other places as well, please also delete the definition.